### DISCUSSION TITLE
Obsidian 字体配置

### DISCUSSION THREAD

**Discussion by giscus** [edony-ink/comments#13] [General]
# Obsidian 字体配置

Obsidian 在 macOS、iOS、iPadOS 设备中字体配置的方法。

https://www.edony.ink/obsidian-fonts-with-css/

**Comment by Jayden-Zhu24** [comment]
作者你好，我想在安卓手机使用obsidian想要更换字体，由于手机系统的限制，所以只能用你文章最后“2、系统未安装字体的配置方式”，想请问一下你在css中src的属性能否引用外部ttf文件，而不是转换为base64 编码数据的 URI，比如“src: url("./font/本墨绪圆-常规.ttf") format("truetype");”，我尝试过这种方式，可是怎么也无法成功。请问是语法问题，还是从本质上根本不可行呢。恳请作者指导一下。

**Reply by edonyzpc** [reply]
理论上语法是支持的，这个问题我建议从两个方面来排查：
1. 你确认一下文件路径（相对路径问题）
2. 文件名改成英文的

P.S. 最近家里有点事情，看评论不是很及时，回复有点晚了

**Reply by edonyzpc** [reply]
参考链接：
https://developer.mozilla.org/zh-CN/docs/Web/CSS/@font-face
